packages/server/README.md

```diff
@@ -93,6 +93,17 @@ If an error is thrown, the HTTP request will be aborted and the provided `body`
 
 This can be used to implement post-processing validation.
 
+#### `options.onIncomingRequest`
+
+`onIncomingRequest` is a middleware function invoked before all handlers (`(req, res, upload) => Promise<res>`)
```

Comment:
This documentation mentions an `upload` argument, which I do not see in the rest of the code. Do you plan on implementing this?

Reply:
Ah that's a copy paste mistake. Because we run `onIncomingRequest` before any handler, no `Upload` is created yet so we can't send it along.